src/styles/_variables.scss

```diff
@@ -24,20 +24,25 @@ $dark-grey: rgb(102, 114, 143);
   border-radius: 10px;
   box-shadow: 3px 3px darken($color, 25%);
 
+  &.disabled {
+    pointer-events: none;
+    color: $dark-grey;
+    background-color: $grey;
+    box-shadow: none;
```

Comment:
could add `cursor: not-allowed;`

Reply:
I've never seen that before. I just tried it, but `cursor: not-allowed` still allows users to hover over the buttons and the cursor changes to "Do not disturb" icon.